tests/test_utils.py

```diff
@@ -121,21 +125,23 @@ def test_correct_eigenvalues_paulis(self, pauli):
 
     @pytest.mark.parametrize("pauli_product", matrix_pairs)
     def test_correct_eigenvalues_pauli_kronecker_products_two_qubits(self, pauli_product):
-       """Test the paulieigs function for two qubits"""
-       assert np.array_equal(pu.pauli_eigs(2), np.diag(np.kron(self.pauliz, self.pauliz)))
+        """Test the paulieigs function for two qubits"""
+        assert np.array_equal(pu.pauli_eigs(2), np.diag(np.kron(self.pauliz, self.pauliz)))
 
     @pytest.mark.parametrize("pauli_product", matrix_pairs)
     def test_correct_eigenvalues_pauli_kronecker_products_three_qubits(self, pauli_product):
-       """Test the paulieigs function for three qubits"""
-       assert np.array_equal(pu.pauli_eigs(3), np.diag(np.kron(self.pauliz, np.kron(self.pauliz, self.pauliz))))
+        """Test the paulieigs function for three qubits"""
+        assert np.array_equal(
+            pu.pauli_eigs(3), np.diag(np.kron(self.pauliz, np.kron(self.pauliz, self.pauliz)))
+        )
 
     @pytest.mark.parametrize("depth", list(range(1, 6)))
     def test_cache_usage(self, depth):
         """Test that the right number of cachings have been executed after clearing the cache"""
         pu.pauli_eigs.cache_clear()
         pu.pauli_eigs(depth)
-        total_runs = sum([2**x for x in range(depth)])
-        assert functools._CacheInfo(depth - 1, depth, 128, depth)== pu.pauli_eigs.cache_info()
+        total_runs = sum([2 ** x for x in range(depth)])
+        assert functools._CacheInfo(depth - 1, depth, 128, depth) == pu.pauli_eigs.cache_info()
```

Comment:
Thanks for getting these formatting ones!

Reply:
Let's all thank the python god for the existence of Black 🥇